llama_stack/distribution/build_container.sh

```diff
@@ -82,13 +82,17 @@ fi
 # Add pip dependencies first since llama-stack is what will change most often
 # so we can reuse layers.
 if [ -n "$pip_dependencies" ]; then
-  add_to_docker "RUN pip install --no-cache $pip_dependencies"
+  add_to_docker << EOF
```

Comment:
Have to add these EOF to make these be added to Dockerfile when running script in non-terminal environment.